app/src/main/assets/task/info/Symptoms.json

```diff
@@ -0,0 +1,4 @@
+{
+  "identifier": "Symptoms",
```

Comment:
Is this how we are loading task info from json?  Where does this get loaded form and referenced?

Reply:
This is loaded in the SageResearchTaskLauncher in the launchTask method.